shell/platform/android/io/flutter/plugin/platform/PlatformViewsController.java

```diff
@@ -406,6 +416,18 @@ private static PointerCoords parsePointerCoords(Object rawCoords, float density)
         return coords;
     }
 
+    private void validateVirtualDisplayDimensions(int width, int height) {
+        DisplayMetrics metrics = mContext.getResources().getDisplayMetrics();
+        if (height > metrics.heightPixels || width > metrics.widthPixels) {
+          StringBuilder errorBuilder = new StringBuilder();
```

Comment:
nit: I think it's more common in out codebase to just use `+` for string concatenation (I believe it is compiled to a StringBuilder anyway).
This should also save the Local import, which initially made me wonder.